src/Google.Cloud.Functions.Invoker/FunctionsStartupAttribute.cs

```diff
@@ -14,25 +14,33 @@
 
 using System;
 
-namespace Google.Cloud.Functions.Invoker.DependencyInjection
+namespace Google.Cloud.Functions.Invoker
 {
     /// <summary>
-    /// Attribute used to declare <see cref="FunctionsStartup"/> types, primarily used to
-    /// configure dependency injection within functions. When multiple types are specified,
-    /// the order in which they are used is undefined.
+    /// Assembly attribute to specify a <see cref="FunctionsStartup"/> to use
```

Comment:
```suggestion
    /// Assembly attribute to specify one or many <see cref="FunctionsStartup"/> to use
```

Reply:
No, this attribute (as in an instance of it) only specifies a single startup type - but it can be applied multiple times. Will add that as another sentence though.